app/models/event_stream.rb

```diff
@@ -35,14 +36,22 @@ class EventStream < ApplicationRecord
 
   after_commit :emit_notifications, :on => :create
 
+  supports_not :timeline, :reason => "Only subclasses can be on a timeline"
+
+  # TODO: Consider moving since this is EmsEvent specific. group, group_level and group_name exposed as a virtual columns for reports/api.
   GROUP_LEVELS = %i(critical detail warning).freeze
 
+  def self.description
+    raise NotImplementedError, _("description must be implemented in a subclass")
```

Comment:
I don't think we need to translate this Exception because it should never be presented to a user (i.e. getting a NotImplementedError raise in production is a bug that needs to be fixed).  Mainly thinking about translators here, as they won't know what this means in order to translate it.